### PR TITLE
Fix tutorial

### PR DESCRIPTION
**Pull request by fluca1978**
Minor fixes to the tutorials.

### CODE REVIEW

File: doc/tutorial/04_prometheus.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ pgagroal-cli -c /etc/pgagroal/pgagroal.conf stop
 pgagroal -c /etc/pgagroal/pgagroal.conf -a /etc/pgagroal/pgagroal_hba.conf
 ```
 
-If you need to specify other configuration files, for example for remote management (see [the related tutorial](https://github.com/pgagroal/pgagroal/blob/main/doc/tutorial/03_remote_management.md)), add them on the `pgagroal~ command line.
+If you need to specify other configuration files, for example for remote management (see [the related tutorial](https://github.com/pgagroal/pgagroal/blob/master/doc/tutorial/03_remote_management.md)), add them on the `pgagroal~ command line.
```
Comment (jesperpedersen) [inline doc/tutorial/04_prometheus.md:46]: ~ -> `

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
There is some text like `[Prometheus](https://prometheus.io/){:target="_blank}` - we should fix those as well

**Comment by jesperpedersen** [comment]
Ok, apparently the syntax is supported in later versions, so lets leave it in

File: doc/tutorial/04_prometheus.md (hunk 1 above)
Comment (fluca1978) [reply doc/tutorial/04_prometheus.md:46]: Thanks, I haven't spot such error!

**Comment by fluca1978** [comment]
I've cross checkd for pending `~` in all the tutorials, seems fine to me now.
Also the Prometheus link problem was a not closed quote.

**Comment by jesperpedersen** [comment]
Merged.

Thanks for your contribution !

Looks like we are ready for 1.5

**Pull request closed without merging**